packages/frontend/app/pods/components/pod/application/top-nav/template.hbs

```diff
@@ -2,14 +2,14 @@
 
   <div class='left-nav'>
     {{#if this.currentUser.isLoggedIn}}
-      <div>
+      <div {{update-document-title unreadMessageCount=this.unreadMessageCount}}>
```

Comment:
Since there is only one arg passed in, do we need to name it?

As a consumer of this API, I don't know if I care that the first arg is called unreadMessageCount. I know that whatever I pass will be the first token in the title.

So, maybe to your point about reusability, maybe the arguments passed should be only positional args, and in the modifier you could do something like 
```ts
let tokens = this.args.positional.join(' | ');
document.title =`${tokens} | emberclear`;
```

That would also allow you to easily pass in the name, as it would just be another token

Reply:
I like that idea